File: 2°SEMESTRE/UC8-prog_orient_obj/JAVA/aula_java/src/AulasJava_thiago/exemplo25_matriz.py

i=0#contador para percorrer linhas 
j=0#contador para percorrer colunas
matriz = []
lin = int(input("Digite a quantidade de linhas: "))
col = int(input("Digite a quantidade de colunas: "))
while i<lin:
    linha = []
    while j<col:
        if i==j:
            linha.append(1)
        else:
            linha.append(0)
        # num =1
        # linha.append(num)
        j=j+1
    matriz.append(linha)
    i=i+1
    j=0
print (matriz)

File: 2°SEMESTRE/UC8-prog_orient_obj/JAVA/aula_java/src/AulasJava_thiago/exemplo25_matriz.java
package AulasJava_thiago;

import java.util.Scanner;

public class exemplo25_matriz {
    public static void main(String[] args){
        Scanner scan = new Scanner(System.in);
        System.out.println("Digite o numeros de linhas: ");
        int linhas=scan.nextInt();
        
        System.out.println("Digite o numeros de colunas: ");
        int colunas=scan.nextInt();
        
        
        int m[][]= new int [linhas][colunas];
        int i, j;
        
        System.out.println("Digite os elementos da mtriz: ");
        for(i=0; i<linhas;i++){
            for(j=0; j<colunas;j++){
                System.out.println("Elemento [" +i+"]["+j+"]: ");
                m[i][j]=scan.nextInt();
            }
        }
        
        for(i=0; i<m.length; i++){
            System.out.printf( "%d linha: ",(i+1) );
            for (j=0; j<m[i].length;j++){
                System.out.printf("%d ",m[i][j]);    
            }
            System.out.println("\n");
        }
        scan.close();


    }
    
}
